pkg/registry/core/podtemplate/strategy.go

```diff
@@ -53,7 +56,13 @@ func (podTemplateStrategy) PrepareForCreate(ctx context.Context, obj runtime.Obj
 // Validate validates a new pod template.
 func (podTemplateStrategy) Validate(ctx context.Context, obj runtime.Object) field.ErrorList {
 	template := obj.(*api.PodTemplate)
-	return corevalidation.ValidatePodTemplate(template)
+	opts := validation.PodValidationOptions{
```

Comment:
use unified helper to get opts, passing `nil` for `old template` (comment applies to all strategy computation of opts)